simulation_parameters/Constants.cs

```diff
@@ -711,6 +754,8 @@ public static class Constants
     /// </remarks>
     public const string AI_TAG_MICROBE = "microbe";
 
+    public const string PROCESSABLE_MICROBE_GROUP = "microbe_process";
```

Comment:
What is this group used for, why not use `PROCESS_GROUP`? Also this constant needs to be placed next to the other group variables (so before AI_TAG_MICROBE)